### PR TITLE
Validate presence of bid amount

### PR DESCRIPTION
**Pull request by jessieay**
* This came up when I was trying to update seed data to get a "needs
  evaluation" auction
* Add indexes to bid foreign keys, too
  https://tomafro.net/2009/08/using-indexes-in-rails-index-your-associations

### CODE REVIEW

**Comment by jessieay** [comment]
Note: i'd merge this *after* https://github.com/18F/micropurchase/pull/1081 since they both have migrations

File: db/seeds.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
   Skill.find_or_create_by(name: 'elm'),
   Skill.find_or_create_by(name: 'a-language-you-never-heard-of')
 ]
+
```
Comment (harrisj) [inline db/seeds.rb]: Looks like we might have some whitespace in here?

File: spec/controllers/bids_controller_spec.rb [hunk 2]
```diff
@@ -7,15 +7,16 @@
   describe '#index' do
     context 'when logged in' do
       it 'should assign auctions that current user have bidded on, presented' do
-        bid = auction.bids.create(bidder_id: current_bidder.id)
+        bid = create(:bid, bidder: current_bidder, auction: auction)
```
Comment (harrisj) [inline spec/controllers/bids_controller_spec.rb]: Yes, much nicer

**Comment by harrisj** [comment]
This is cool, I'd like to merge this one, but I have concerns about #1081

**Comment by harrisj** [comment]
Ooops, you need to rebase this one now. Sorry!

**Comment by jessieay** [comment]
@harrisj rebased! silly db/schema.rb..

**Comment by harrisj** [comment]
LGTM Merging

**Pull request merged**